login_dev.ps1

```diff
@@ -0,0 +1,10 @@
+python3 ..\aws-cli-tools\samlauth\samlauth.py -a NVAdev
+python3 .\aws.py | ForEach-Object {
+    #Expecting all result to be $env:NAME=VALUE for variables that need to be set
+    $a = $_ -replace '\$env:(\w+)=(.*)', '$1;$2' -split ';'
+
+    #Setting variables at process-level. Can be replaced with "User" and "Machine" for permanent variables
+    [System.Environment]::SetEnvironmentVariable($a[0], $a[1], "Process")
+}
+[System.Environment]::SetEnvironmentVariable("CYPRESS_AWS_USER_POOL_ID", "eu-west-1_ZgmwmNtMp", "Process")
+[System.Environment]::SetEnvironmentVariable("CYPRESS_AWS_CLIENT_ID", "5lvkqvapt1bbbnalce5qu3agbi", "Process")

```

Comment:
hva er denne fila? skal den være med?
(samme for `login_sandbox.ps1)`

Reply:
Skal ikke være med, lokalt windows-script for å kjøre Cypress